app/view_models/Summary.scala

```diff
@@ -32,13 +33,16 @@ case class Summary(
   taxableGains: Amount,
   cgTaxPerCurrencyUnit: Amount,
   nicsAndTaxPerCurrencyUnit: Amount,
+  incomeAfterTaxAndNics: Amount,
   totalCgTaxRate: Rate,
-  nicsAndTaxRate: Rate,
+  nicsAndTaxRate: Amount,
```

Comment:
Should this change affect SA and PAYE?

Reply:
:) You've just found a bug I've recently introduced. PAYE needs this as an amount, SA needs it as a rate. We will need to add another field in the view specifically for PAYE. Thx.